tests/Doctrine/Tests/Common/Annotations/AnnotationReaderTest.php

```diff
@@ -165,4 +165,22 @@ public function testPHPCodeSnifferAnnotationsAreIgnored()
 
         self::assertEmpty($reader->getClassAnnotations($ref));
     }
+
+    public function testFunctionsAnnotation() : void
+    {
+        $reader = $this->getReader();
+        $ref = new \ReflectionFunction('Doctrine\Tests\Common\Annotations\Fixtures\foo');
+
+        $annotations = $reader->getFunctionAnnotations($ref);
+        self::assertInstanceOf(Fixtures\Annotation\Autoload::class, $annotations[0]);
```

Comment:
Can you please also check for the amount of elements in that annotations array with `assertCount`?